01 - Community/People/timhor.md

```diff
@@ -1,12 +1,12 @@
 ---
 aliases:
-- Tim Hor
+- Andrew Brown & Tim Hor
```

Comment:
I'd revert this one. We need a better way to handle multiple authors from json files.

Reply:
Done